backend/src/routers/gameRouter.ts

```diff
@@ -164,6 +173,13 @@ gameRouter.post(
 		);
 
 		res.status(200).json({ message: 'Game added' });
+		} catch (e) {
+			if (e instanceof Error) {
+				res.status(400).json({ message: "Something went wrong adding the game" });
+			} else {
+				res.status(500).json({ message: "Uwu oopsie woopsie, the devs made a fucky wucky! Sowwy" });
```

Comment:
This message. This is perfection. No code in this repository will ever top the quality of this single line right here. We might as well archive the entire repo at this point, we're all done here.